configs/fastscnn/fast_scnn_4x8_80k_lr0.12_cityscapes.py

```diff
@@ -0,0 +1,64 @@
+_base_ = [
+    '../_base_/models/fast_scnn.py', '../_base_/datasets/cityscapes.py',
+    '../_base_/default_runtime.py'
+]
+crop_size = (512, 1024)
+cudnn_benchmark = True
+# model training and testing settings
+train_cfg = dict()
+test_cfg = dict(mode='whole')
```

Comment:
Move these to `configs/_base_/models/fast_scnn.py`.

Reply:
Moved.